msvc/ut-build-early.props

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Starboard OneCore Overrides">
+    <_SB_Local_VCInstallDir Condition="'$(_SB_Local_VCInstallDir)' == ''">$(Registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\14.0\Setup\VC@ProductDir)</_SB_Local_VCInstallDir>
```

Comment:
I agree with Pradip here. THis is a bit scary. We might want to talk to VS folks to see what else we could do?

---
In reply to: [74666494](https://github.com/Microsoft/WinObjC/pull/874#discussion_r74666494) [](ancestors = 74666494,74666079)